lms/djangoapps/certificates/tests/test_webview_views.py

```diff
@@ -1079,6 +1080,8 @@ def test_certificate_custom_language_template_with_org_mode_and_course_key(self,
         Tests custom template search and rendering.
         This test should check template matching when org={org}, course={course}, mode={mode}.
         """
+        DarkLangConfig(released_languages='es-419, fr', changed_by=self.user, enabled=True).save()
```

Comment:
nit: These tests now have a dependency on DarkLangConfig language fallback to be working correctly. The released languages list should be testings with 'es, fr'